source/change-streams/change-streams.rst

```diff
@@ -47,6 +47,7 @@ An error is considered resumable if it meets any of the following criteria:
   network error)
 
 - *any* server error response from a getMore command excluding those
+  containing the error label `NonRetryableChangeStreamError` and those
```

Comment:
@ShaneHarvey: Is this something you intended to test? IIRC, we the transaction tests assert error labels after a `failCommand` fail point but I'm not sure that we can configure the fail point to return a specific error code. In the case of the txn tests, I think we're relying on existing driver or server behavior to apply an error label for an error code.

Reply:
The server added this new error label in https://jira.mongodb.org/browse/SERVER-39675 so we can write a spec test against >=4.1.10, see [my comment above](https://github.com/mongodb/specifications/pull/528#issuecomment-478098724).